app/policies/portfolio_policy.rb

```diff
@@ -17,8 +17,11 @@ def update?
     rbac_access.update_access_check
   end
 
-  alias share? update?
-  alias unshare? update?
+  def share?
+    rbac_access.admin_access_check("portfolios", "update")
```

Comment:
@eclarizio Do we need spec changes in portfolio_policy_spec to account for this new logic.

Reply:
Woops, yeah, was making this and #682 at the same time and missed them.